kafka-client/src/main/java/dev/responsive/kafka/api/SharedClients.java

```diff
@@ -0,0 +1,95 @@
+package dev.responsive.kafka.api;
+
+import dev.responsive.db.CassandraClient;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.ScheduledExecutorService;
+import org.apache.kafka.clients.admin.Admin;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Basic container class for session clients and other shared resources
+ */
+public class SharedClients {
```

Comment:
just to make sure I understand, this isn't part of the "public" API either, right? can we also indicate that by making it package private?

Reply:
Yep, that was an oversight on my part. Good catch

Reply:
Ended up just moving it to the `clients` package, hope that's ok too